OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -74,11 +74,6 @@ public override Activity Tick(Actor self)
 				self.QueueActivity(mobile.MoveTo(moveTo, 1));
 				self.SetTargetLine(Target.FromCell(self.World, moveTo), Color.Gray, false);
 
-				// TODO: The harvest-deliver-return sequence is a horrible mess of duplicated code and edge-cases
```

Comment:
>  If we're going to remove these notifications then we should go further and remove the whole code branch.

I'd prefer to remove only the "get out of the way" stuff, not the whole branch, because I have a follow-up that is using that branch.